setup.py

```diff
@@ -29,7 +29,7 @@
 install_requires = [
     "numpy>=1.19",  # 1.19 required by tensorflow 2.6
     "pandas>1.0.3,<2",
-    "scipy>=1.0.0,<2,!=1.11.0",
+    "scipy>=1.0.0,<2,!=1.11.0,!=1.11.1",
```

Comment:
I have the hope that they fix it, in future versions rather than constraining us to <1.11.2. It seems that they didn't really fix the problem in 1.11.1, so they might need to yank it as well o_o

Let's see if the tests pass, I'll contact them.

Reply:
I am not sure it is a bug though. It could be a feature :-)
Their change probably makes sense because now the symbol is consistent with numpy. Probably we need to switch to the new symbol in the code.

Reply:
@anargyri you are saying that instead of `test_scores = self.user_affinity[user_ids, :].dot(self.item_similarity)` we should do `test_scores = self.user_affinity[user_ids, :]*self.item_similarity`? so maybe dot is deprecated?